devtools/maven/pom.xml

```diff
@@ -102,6 +102,12 @@
             <version>2.14.6</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.freemarker</groupId>
+            <artifactId>freemarker</artifactId>
+            <version>2.3.28</version>
```

Comment:
What do you think about moving the version number to `quarkus-build-parent`'s DM section?

Reply:
+1

Reply:
Fixed in 344226776dbd42ba63a4c2163d3ba9955cc232cf